requirements.txt

```diff
@@ -3,3 +3,6 @@ wagtail==1.0b2
 django-hashers-passlib==0.1
 django-storages-redux==1.2
 boto3
+django-contrib-comments==1.6.1
+django-comments-xtd==1.4b2
+django-longerusernameandemail
```

Comment:
Is this really needed? Do the tests pass without it? I'm 99% sure it does nothing at all in django 1.8.